features/steps/gradle_plugin_steps.rb

```diff
@@ -59,18 +59,40 @@
   requests = get_build_requests
   assert_equal(request_count, requests.length, 'Wrong number of build API requests')
   RequestSetAssertions.assert_requests_match requests, data_table
+
+  requests.each do |request|
+    valid_build_api?(request[:body])
+  end
 end
 
 Then('{int} requests are valid for the android mapping API and match the following:') do |request_count, data_table|
   requests = get_android_mapping_requests
   assert_equal(request_count, requests.length, 'Wrong number of mapping API requests')
   RequestSetAssertions.assert_requests_match requests, data_table
+
+  requests.each do |request|
+    valid_android_mapping_api?(request[:body])
+  end
 end
 
 Then('{int} requests are valid for the android NDK mapping API and match the following:') do |request_count, data_table|
   requests = get_android_ndk_mapping_requests
   assert_equal(request_count, requests.length, 'Wrong number of NDK mapping API requests')
   RequestSetAssertions.assert_requests_match requests, data_table
+
+  requests.each do |request|
+    valid_android_ndk_mapping_api?(request[:body])
+  end
+end
+
+Then('{int} requests are valid for the android Unity NDK mapping API and match the following:') do |request_count, data_table|
+  requests = get_android_ndk_mapping_requests
```

Comment:
Will this become `get_android_unity_ndk_mapping_requests` at some point?

Reply:
Yep - updated accordingly